content/docs/addons-test-utils.md

```diff
@@ -221,7 +221,7 @@ scryRenderedDOMComponentsWithClass(
 )
 ```
 
-Finds all DOM elements of components in the rendered tree that are DOM components with the class name matching `className`.
+尋找已 render 的 tree 中 component 的所有 DOM element 裡，class 名稱符合 `className` 的 DOM component。
```

Comment:
```suggestion
在已經被 render 的 tree 中尋找所有 DOM element，class 名稱符合為 `className` 的 DOM component。
```

Reply:
Skipping `components` makes the sentence easier to read.
But I think it can be changed into..

```suggestion
在已經被 render 的 tree 中尋找所有 DOM element，回傳 class 名稱符合 `className` 的 DOM component。
```